python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/tunnel-interfaces.j2

```diff
@@ -28,11 +28,15 @@
 {%         set shutdown = tunnel_interface.shutdown | arista.avd.default('-') %}
 {%         set nat_profile = tunnel_interface.nat_profile | arista.avd.default('-') %}
 {%         set tunnel_mode = tunnel_interface.tunnel_mode | arista.avd.default("-") %}
-{%         set row_source_interface = tunnel_interface.source_interface | arista.avd.default("-") %}
+{%         if tunnel_interface.source_interface is arista.avd.defined %}
+{%             set row_source = tunnel_interface.source_interface %}
+{%         else %}
+{%             set row_source = tunnel_interface.source | arista.avd.default("-") %}
```

Comment:
This could have been a one-liner with `set row_source = tunnel_interface.source_interface | default(tunnel_interface.source, "-")`